docs/source/contributing/jupyter_style.md

```diff
@@ -58,6 +169,123 @@ Choose a category from [existing categories](https://github.com/pymc-devs/pymc/w
+    and then either the `extra_dependencies` or the `pip_dependencies` and `conda_dependencies`.
+    In addition, there is also an `extra_install_notes` to include custom text inside the dropdown.
+
+    * notebook metadata can be edited from the menubar `Edit` -> `Edit notebook metadata`
+      in the dropdown
+
+      This will open a window with json formatted text that might look a bit like:
+
+      ::::{tab-set}
+      :::{tab-item} No myst_substitutions
+
+      ```json
+      {
+        "kernelspec": {
+          "name": "python3",
+          "display_name": "Python 3 (ipykernel)",
+          "language": "python"
+        },
+        "language_info": {
+          "name": "python",
+          "version": "3.9.7",
+          "mimetype": "text/x-python",
+          "codemirror_mode": {
+            "name": "ipython",
+            "version": 3
+          },
+          "pygments_lexer": "ipython3",
+          "nbconvert_exporter": "python",
+          "file_extension": ".py"
+        }
+      }
+      ```
+      :::
+
+      :::{tab-item} extra_dependencies key
+
+      ```{code-block} json
+      :emphasize-lines: 19-21
+      {
+        "kernelspec": {
+          "name": "python3",
+          "display_name": "Python 3 (ipykernel)",
+          "language": "python"
+        },
+        "language_info": {
+          "name": "python",
+          "version": "3.9.7",
+          "mimetype": "text/x-python",
+          "codemirror_mode": {
+            "name": "ipython",
+            "version": 3
+          },
+          "pygments_lexer": "ipython3",
+          "nbconvert_exporter": "python",
+          "file_extension": ".py"
+        },
+        "myst_substitutions": {
+          "extra_dependencies": "bambi seaborn"
+        }
+      }
+      ```
+      :::
+
+      :::{tab-item} pip and conda specific keys
+      ```{code-block} json
+      :emphasize-lines: 19-22
+      {
+        "kernelspec": {
+          "name": "python3",
+          "display_name": "Python 3 (ipykernel)",
+          "language": "python"
+        },
+        "language_info": {
+          "name": "python",
+          "version": "3.9.7",
+          "mimetype": "text/x-python",
+          "codemirror_mode": {
+            "name": "ipython",
+            "version": 3
+          },
+          "pygments_lexer": "ipython3",
+          "nbconvert_exporter": "python",
+          "file_extension": ".py"
+        },
+        "myst_substitutions": {
+          "pip_dependencies": "graphviz",
+          "conda_dependencies": "python-graphviz",
+        }
+      }
+      ```
+
+      The pip and conda spcific keys overwrite the `extra_installs` one, so it doesn't make
+      sense to use `extra_installs` if using them. Either both pip and conda substitutions
+      are defined or none of them is.
+      :::
+      ::::
+
+1.  include the warning and installation advise template with the following markdown:
```

Comment:
Is there a position preferred for this markdown, for example after the title or  after the imports?

Reply:
good point, I'd say right before imports so libraries can be installed and then imported